src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -269,4 +273,27 @@ public static AssemblyResultSet assembleReads(final AssemblyRegion region,
             throw e;
         }
     }
+
+    /**
+     * @return the default set of read filters for HC (includes the MQ threshold filter) or M2 (does not include the MQ filter)
+     */
+    public static List<ReadFilter> makeStandardReadFilterList(final boolean includeMQThresholdFilter) {
```

Comment:
@droazen will do. Thanks for reviewing the changes. @davidbenjamin please wait for my next commit.